scripts/assign.py

```
@@ -207,34 +207,110 @@ def main():
+                                .get("timelineItems", {})
+                                .get("nodes", [])
+                            )
+
+                            for item in timeline_items:
+                                source = item.get("source", {})
+                                if source and "state" in source and source["state"] == "OPEN":
+                                    pr_number = source.get("number")
+                                    print(f"Found open PR #{pr_number} linked to issue #{current_issue_number}")
+                                    has_pr = True
+                                    break
+                    except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)